### DISCUSSION TITLE
2022-02-02 Weekly Meeting 41 Agenda

### DISCUSSION THREAD

**Discussion by antpb** [omigroup/omigroup#177] [Weekly Meetings]
Weekly Meeting 41 is happening on Wednesday Feb 2nd, 2022 from 9:00 AM PDT - 10:00 AM PDT (17:00 UTC)

You can add it to your calendar [here](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t).
 
We'll be meeting again in the omi-weekly-meeting channel of the [AngellXR/OMI Discord](https://discord.gg/NJtT9grz5E).

## Who we are and why we're here:

The Open Metaverse Interoperability Group (OMI) is focused on bridging virtual worlds by designing and promoting protocols for identity, social graphs, inventory, and more. Our members include businesses and individuals working towards this common goal. 

If you’re an observer or here for the first time, this is a meeting where people who work on the OMI group initiatives discuss updates and contribution opportunities for current projects and goals. This is your chance to jump in and help!

We welcome you to introduce yourself in #omi-meeting-chat as we dive into the agenda items. There will be an open floor in the last 15 minutes of the meeting (agenda permitting) where we can discuss any items you feel are relevant to the current or near future OMI goals.

## Agenda

- Volunteer to take notes? Volunteer to record? 
- Welcome newcomers

### Announcements and Updates

  - Election is Today (scroll to bottom): https://github.com/omigroup/omigroup/issues/174
  - Proposed Change to Voting Rule : https://github.com/omigroup/omigroup/issues/175
  - Proposed minimum requirements for a working group to operate: https://github.com/omigroup/omigroup/issues/176 
 - Updated Working agreement template: https://github.com/omigroup/omigroup/issues/68
 - Omi-chan working group kicked off Saturday: https://github.com/omigroup/omi-chan
 - gLTF group updates 
 - WASM group updates 
 - backlog refinement kicked off last week (that's it no additional updates needed)

 - Open Floor!
 
Please leave any additional agenda items below in the comments. Thanks!

**Comment by mrmetaverse** [comment]
Wow! Check out this snapshot of some OMI-Chan avatar designs from the new [OMI-Chan working group](https://github.com/omigroup/omi-chan/discussions/13)!

**Comment by mrmetaverse** [comment]
Today we can vote to extend the existing terms by 90 days and commit to continuing to discover our new structure, or we can have the election as scheduled and commit to a new structure within 90 days. I defer to the group to make this decision with two votes if needed: 

Vote 1: Vote to extend existing chairs by 90 days 
Vote 2 (if vote 1 fails): Have the election as usual. 

I don't see any nominations so I assume the group wants the same chairs but we are all committed to dividing our responsibilities and creating new roles. I know I'm not alone when I say I want to do the right thing and find the best structure for all rather than just going through the motions. 

I will be on mountain today and may have shoddy service. I am honored to have served and to have a group like you to meet with, and would be honored to continue serving as chair. But, I will be committed to finding the right structure in the next 90 days, and I am tracking this effort with [this milestone: Future Structure of OMI](https://github.com/omigroup/omigroup/milestone/2)*

**Comment by mrmetaverse** [comment]
We are tracking the new structure effort with [this milestone](https://github.com/omigroup/omigroup/milestone/2)
![image](https://user-images.githubusercontent.com/63426722/152203754-97d0df6d-ab55-4e70-90f8-98411772a7c2.png)

**Comment by mrmetaverse** [comment]
VOTE 1: Extend existing chairs for 90 days and commit to discovering new structure 
![image](https://user-images.githubusercontent.com/63426722/152205181-24028d2a-3417-4e42-9552-5ca30bf25f10.png)

**Comment by mrmetaverse** [comment]
Vote 2: Vote to change voting to rules to accommodate attendance of working groups - PASSED
![image](https://user-images.githubusercontent.com/63426722/152206570-4ab91780-c285-4541-b799-b7f65af0ad7b.png)

**Reply by mrmetaverse** [reply]
TO DO: Create pull request

**Reply by robertlong** [reply]
Done https://github.com/omigroup/omigroup/pull/179

**Comment by mrmetaverse** [comment]
We discussed minimum requirements for a working group, and the provision of templates for things like readme's and working agreements. The group will continue to discuss async and pick up the convo next week. What do we want the minimum reqs to be? 

Robert also suggested making these issues PR's instead of issues.

**Comment by mrmetaverse** [comment]
OMI-Chan kicked off this past weekend. We discussed their work and reviewed their readme and working agreement as exemplars of a good kickoff.

**Comment by mrmetaverse** [comment]
gLTF group has great updates. One including that two members of the group attended a Khronos group today to share our work and process.

**Reply by mrmetaverse** [reply]
Mauve also has made some significant contributions to the gLTF group including colliders, and other extensions.

**Comment by mrmetaverse** [comment]
WASM Group: 
- had a good discussion last meeting 
- working on coordinating around a minimum viable piece of estate transferred between a runtime and WebAssembly context 
- reviewed what webassembly is, we are trying to explore a way various runtimes can be scriptable using a format called WebAssembly. 
- This takes a game engine like Unity or Godot and embeds a runtime that can interpret a format that would be common across all of these runtimes to create some portable script or business logic that can be usable by any game engine and some sliver of the metaverse can exist. 
- MVProject is some boolean state that can set collide to false in some object in the game universe. 
- group is going off into their corners and experimenting to implement tiny demos

**Reply by mrmetaverse** [reply]
Proposed async WASM jam to experiment together: https://github.com/omigroup/omi-wasm-group/discussions/19

**Reply by mrmetaverse** [reply]
Proposed announcing on twitter and maybe syndicating with a twitter spaces. Timeline to review samples is about 2 weeks from yesterday.

**Comment by mrmetaverse** [comment]
Rick shared his notes from today as well: 
![image](https://user-images.githubusercontent.com/63426722/152211826-89025939-55f3-4b38-b11a-390258c23474.png)
